catalog/package.json

```diff
@@ -130,7 +130,6 @@
     "remarkable": "^1.7.4",
     "reselect": "^4.0.0",
     "sanitize.css": "^11.0.1",
-    "tslib": "^2.2.0",
```

Comment:
this should be kept under "dependencies", bc it's a runtime dependency

Reply:
But there are no actual "tslib" imports in our code. Is this runtime dependency injected during the building step? "build" directory doesn't have "tslib" either